scripts/publish.sh

```diff
@@ -4,9 +4,9 @@ set -x
 set -e
 set -o pipefail
 
-if [[ -z $NPM_TOKEN ]]; then
-  echo "Notice: NPM_TOKEN environment variable not set. Running 'npm publish --dry-run'."
-  npm publish --dry-run
+if [[ -z $YARN_NPM_AUTH_TOKEN ]]; then
+  echo "Notice: 'npm-token' not set. Running 'yarn pack --dry-run'."
```

Comment:
I updated this message to use the user-facing parameter name `npm-token`, since it had to change anyway